app.py

```diff
@@ -37,7 +37,8 @@
 @click.option("--force-sync", is_flag=True, help="Perform force sync of documents.", envvar="THOTH_SYNC_FORCE_SYNC")
 @click.option("--graceful", is_flag=True, help="Continue on any error during the sync.", envvar="THOTH_SYNC_GRACEFUL")
 @click.option("--debug", is_flag=True, help="Run in a debug mode", envvar="THOTH_SYNC_DEBUG")
-def sync(force_sync: bool, graceful: bool, debug: bool) -> None:
+@click.option("--only-sync", help="Perform sync of specific class of documents", envvar="THOTH_SYNC_ONLY")
+def sync(force_sync: bool, graceful: bool, debug: bool, only_sync: Optional[str] = None) -> None:
```

Comment:
Would it make sense to introduce a different parameter name and have the classes comma separated? ``--only-sync`` sound like a flag and the sync job should sync so ``--only-sync`` sounds strange to me.

Reply:
Added `THOTH_SYNC_DOCUMENT_CLASSES`